node_modules/credscan-pkg/install.js

```diff
@@ -0,0 +1,23 @@
+const exec = require('child_process').exec;
```

Comment:
rename the file to pre_install_validation.js

Reply:
Renaming done